contracts/ColonyFunding.sol

```diff
@@ -213,19 +217,19 @@ contract ColonyFunding is ColonyStorage, DSMath {
 
     pots[0].balance[payout.tokenAddress] = sub(pots[0].balance[payout.tokenAddress], reward);
 
-    userRewardPayoutCount[msg.sender] += 1;
-
-    // TODO: Unlock user tokens
+    userRewardPayouts[msg.sender][_payoutId] = true;
+    tokenLocking.unlockTokenForUser(address(token), msg.sender);
```

Comment:
So this `unlockTokenForUser` call checks if the tokens are locked and throws otherwise. I don't think the `usersTokensLocked` call is needed therefore.